src/cytodataframe/frame.py

```
@@ -1676,6 +1793,681 @@ def process_image_data_as_html_display(
+        self: CytoDataFrame_type,
+        row: Any,
+        column: Any,
+    ) -> Tuple[np.ndarray, Tuple[int, int, int]]:
+        cache = self._custom_attrs.get("_volume_cache", {})
+        cache_key = f"{row}::{column}"
+        if cache_key in cache:
+            return cache[cache_key]
+
+        try:
+            value = self.loc[row, column]
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)